packages/adblocker-electron/package.json

```
@@ -34,7 +34,7 @@
     "test": "nyc mocha --config ../../.mocharc.js"
   },
   "peerDependencies": {
-    "electron": "11.x || 12.x || 13.x"
+    "electron": "^11.0.0"
```

I think this needs to be either * or x, otherwise it will not allow major version updates.
```suggestion
    "electron": "*"
```

---

I just check this on https://semver.npmjs.com/ and you are correct, so I experimented with the syntax, and it turns out we can do `>11` to specify that everything starting at 11 is okay, including major versions.

---

The reason I used `>11` instead of `>10`, which is in line with current, is that Electron 11 has reached end of life on the 31st of August.

---

Just to clarify, is the intent to also include `11` or to exclude it and only support from 12 onward? If the former, then should it be `>=11`?

---

The intent here is to exclude 11, and specify it to mean 12 and above, so we could also use `>=12`. Or in case you still want to support 11, it's indeed correct to use `>=11`. Which version has your preference?

---

Both are ok, I just wanted to confirm my understanding. I thought from the previous comment that you meant to exclude 10 but include 11. Thank you!